packages/backend/src/services/leasing-service/adapters/core-adapter.ts

```diff
@@ -128,6 +133,12 @@ const getTenantByContactCode = async (
 
     return { ok: true, data: result.content }
   } catch (err) {
+    if (err instanceof AxiosError && err.response?.status === 500) {
+      if (err.response.data.type) {
+        return { ok: false, err: err.response.data.type, statusCode: 500 }
+      }
+    }
+
     return { ok: false, err, statusCode: 500 }
```

Comment:
I guess this could be condensed like so
```suggestion
    return { ok: false, err: err.response.data.type || err, statusCode: 500 }
```

But I feel like I could be missing something 😅

Reply:
Typescript gillar inte det... AxiosError har response, inte err